Documentation/network/kubernetes/ciliumcidrgroup.rst

```diff
@@ -0,0 +1,60 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _CiliumCIDRGroup:
+
+***************************
+CiliumCIDRGroup
+***************************
+
+.. note::
+
+    **Beta Feature:** CiliumCIDRGroup is currently in beta. We welcome your feedback
+    and encourage you to report any issues by filing a GitHub issue.
```

Comment:
nit: We could probably replace this with the canonical beta.rst include instead:
```suggestion
.. include:: ../beta.rst
```

Feel free to follow up on that in a subsequent PR if you like.